solidity/dashboard/src/lib/keep/coverage-pool/index.js

```diff
@@ -270,6 +273,10 @@ class CoveragePoolV1 {
       withdrawalTimeout,
     }
   }
+
+  hasRiskManagerOpenAuctions = async () => {
+    return await this.riskManagerV1Contract.makeCall("hasOpenAuctions")
+  }
```

Comment:
Can we add a unit test for this new function?

Reply:
27e566d66d3f2927b7ed6dc7672c137ba2db746e